exec/vector/src/main/java/org/apache/drill/exec/vector/accessor/writer/OffsetVectorWriterImpl.java

```diff
@@ -290,7 +290,7 @@ public void preRollover() {
     // rows. But, this being an offset vector, we add one to account
     // for the extra 0 value at the start.
 
-    setValueCount(vectorIndex.rowStartIndex() + 1);
+    setValueCount(vectorIndex.rowStartIndex());
```

Comment:
` But, this being an offset vector, we add one to account for the extra 0 value at the start.` - Should this comment be updated after the change?